src/response/chi0.jl

```diff
@@ -312,9 +312,16 @@ end
                                              kwargs_sternheimer...)
         end
     end
+
+    # pad δoccupation TODO is this correct?
```

Comment:
Yes we essentially assume they don't participate in the physics at all (their occupation stays zero). Feel free to remove the comment. Maybe we should even remove the occ_occ or δocc array and replace it by the δoccupation straight from the start (to avoid the allocation and copy here).